src/secp256k1.c

```diff
@@ -215,6 +222,20 @@ void secp256k1_scratch_space_destroy(const secp256k1_context *ctx, secp256k1_scr
     secp256k1_scratch_destroy(&ctx->error_callback, scratch);
 }
 
+/* Mark memory as no-longer-secret for the purpose of analysing constant-time behaviour
+ *  of the software. This is setup for use with valgrind but could be substituted with
+ *  the appropriate instrumentation for other analysis tools.
+ */
+static SECP256K1_INLINE void declassify(const secp256k1_context* ctx, void *p, size_t len) {
```

Comment:
It's static inline, so it shouldn't matter. I don't care one way or another, utility functions don't seem to be.

Reply:
I suggest moving this to `util.h` then.

Reply:
okay.

Reply:
un-okay:  I changed this and blew CI up... ctx is an opaque type in util.h when util.h is being used in benchmarks and such, causing them to fail to compile.  I didn't like hoisting out the ctx-> dereference because I want that compiled out when not compiled with DVALGRIND, and changing things so util.h could see the non-opaque type made a mess when I briefly attempted it.   When I said okay, I didn't really care either way. I found reasons to care.